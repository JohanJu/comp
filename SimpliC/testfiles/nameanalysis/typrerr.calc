int main()
{
    int a = 5;
    if(a){
    	a = a+1;
    }
}