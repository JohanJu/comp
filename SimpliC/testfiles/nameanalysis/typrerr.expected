Error at line 4: wrong type: 
